src/app/index.ts

```diff
@@ -18,9 +20,20 @@ export const app = async (): Promise<void> => {
     res.status(500).json({ message: err.message });
   });
 
-  app.listen(port, () => {
-    console.log(`App is running on port ${port}`);
-  });
+  const uri: string = `mongodb://localhost:27017/wave_coding`;
```

Comment:
@hsedaqat1378 please move this one to the `.env` file and update `.env.example` file

Reply:
I change this and thanks for tell me about this i know i'm write dirty code